typescript/src/lib/enclave_context/enclave_context.ts

```diff
@@ -0,0 +1,31 @@
+import {APIContainerContext} from "../api_container_context/api_container_context";
+
+export class EnclaveContext {
+    private readonly enclaveID: string;
```

Comment:
Go's naming convention says "uppercase acronyms, like `ID`" so this is correct in Go, but Typescript/Java naming convention says "lowercase acronyms". So we'd want these to be `enclaveId`, `networkId`, `networkCidr`, etc.

(I know, it's annoying :( but we def want to conform to the conventions of the language we're writing in)